repository/src/main/java/com/epam/esm/repository/builder/OrderQueries.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.esm.repository.builder;
+
+public final class OrderQueries {
```

Comment:
1) Why is this class in the builder package?

Reply:
fixed